external/NETNative/project.json.template

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.NETNative": "1.1.0-beta-24911-00"
+      }
+    }
+  },
+  "runtimes": {
+    "{RID}": {}
```

Comment:
originally I thought that we would use this for both uap and uapaot and I didn't know we would use the targeting pack which is why I did it like this. I can go back and change it if you want me to.